Jenkinsfile

```diff
@@ -12,7 +12,7 @@
 
 // To use a test branch (i.e. PR) until it lands to master
 // I.e. for testing library changes
-//@Library(value="pipeline-lib@your_branch") _
+@Library(value="pipeline-lib@jenkins_plugin_for_cart_valgrind")
```

Comment:
Does this PR still need the pipeline-lib change?  From https://github.com/daos-stack/pipeline-lib/pull/267 it looks like there's a regexp change which I'm not sure is related and a default value for a parameter where presumably you could pass in a value, so there's no need to change the default in pipeline-lib?

Reply:
Correct -- these two commits obviate [the pipeline-lib branch](https://github.com/daos-stack/pipeline-lib/pull/267):  f37ab9b357b070998217ad4ffd0d491b8c7109ea d3d9a671d80ee5cdab96fa930d4f3a5c25cd7220.  (`echo -n` and `valgrind_pattern` as argument to the `valgrindReportPublish`.)